### DISCUSSION TITLE
How do I start if I am new to this?

### DISCUSSION THREAD

**Discussion by prgorasiya** [swiftyfinch/Rugby#50] [Q&A] [answered]
Hi there,

Apologies for asking such a basic question.
It would be really helpful if I can get some more detailed doc to implement Rugby.
As a newbie I am not familiar with several concepts, like installing/using Mint, creating Mintfile etc.
Can anyone help me in this?

Thanks in advance.

**Comment by swiftyfinch** [comment, marked as answer]
Hello! You can use `brew` as the easiest way to install `Mint`.
```sh
brew install mint
```
More information [here](https://github.com/yonaskolb/Mint#homebrew).

Then try to install Rugby as described [here](https://github.com/swiftyfinch/Rugby#quick-start-with-mint-):
```sh
mint install swiftyfinch/rugby
```

I think that's all. Then just run `rugby` command :)
Do you have any questions? You can attach some screenshots if you encourage some troubles.
Also, you can describe some details of your installation process and we can add it to Rugby docs.

**Reply by swiftyfinch** [reply]
@prgorasiya Hello! Did it help you?

**Reply by swiftyfinch** [reply]
@prgorasiya Also, I've prepared some demo videos:
🎬 Installation Demo #71
🎬 Basic Usage Demo #72

**Reply by prgorasiya** [reply]
Sorry for the late response. Thanks @swiftyfinch I got it setup on my machine.

**Reply by ghost** [reply]
Hi @swiftyfinch 
Thanks for developing this amazing Rugby.

I'm looking for the installation via SPM.
Could you kindly support in future?

BTW, hope ko-fi support Apple Pay payment.

**Reply by ghost** [reply]
Oh I just realize mint.
Sorry about previous comment.

**Reply by swiftyfinch** [reply]
@Wei-Pinkoi Unfortunately, I couldn't add Apple Pay payment support yet.
Maybe there isn't any way for my country. Or I couldn't find one.
